client/src/navigation/MainNavigator.js

```diff
@@ -43,11 +44,21 @@ const MainTabNavigator = TabNavigator(
   {
     Home: {
       screen: HomeNavigator,
-      navigationOptions: {
+      navigationOptions: ({ navigation }) => ({
         tabBarLabel: 'Home',
         // eslint-disable-next-line react/prop-types
         tabBarIcon: ({ tintColor }) => <Icon size={34} name="home" color={tintColor} />,
-      },
+        tabBarOnPress() {
+          const resetAction = NavigationActions.reset({
```

Comment:
could you not do this only if the tab is already selected?

Reply:
I played around with detecting that and its possible.

But found that users would be on another tab and want to go home so press home, and it wouldnt take them home, and that was confusing.